Allow pushing tokens to the back by pressing 'z' while hovered over a token

cs96and opened the pull request:
Allow pushing tokens to the back by pressing 'z' while hovered over a token

cs96and committed:
Allow pushing tokens to the back by pressing 'z' while hovered over a token

theripper93 commented:
I will accept the pr after a couple fixes:

1: you can access the currently hovered token with `canvas.tokens.hover` no need to add a custom function
2: Refactor the keybinding to use the foundry API for keybindings so it's also reconfigurable
3: Come up with a method that does not involve setting the flag on the token, instead use some method that only changes the order locally.

cs96and committed:
Send to back improvements...
- Use foundry key handling
- Use local z-index override instead of setting teh zIndex flag.
- Use canvas.tokens.hover.

cs96and commented:
I've updated it with your suggestions.

theripper93 commented:
Refactored the code a bit, changed some things - pushing to public
